src/main/java/com/checkmarx/sdk/dto/ScanResults.java

```diff
@@ -185,7 +185,7 @@ public boolean isAstResults(){
         return Optional.ofNullable(getAstResults()).isPresent() &&  Optional.ofNullable(getAstResults().getResults()).isPresent();
     }
 
-    public boolean isSastRestuls(){
+    public boolean isSastResults(){
```

Comment:
I assume there are also changes in cxflow to make it compatible.
let's do it after pull request  #8 is done